ocs_ci/ocs/constants.py

```diff
@@ -395,3 +398,4 @@
 # Deployment constants
 OCS_CSV_PREFIX = 'ocs-operator'
 LOCAL_STORAGE_CSV_PREFIX = 'local-storage-operator'
+INTERNAL_MIRROR_PEM_FILE = "ops-mirror.pem"
```

Comment:
Do we have somewhere documented how user should get it and where to put?

Reply:
Done, Added a doc line for AWSUPI with RHEL workers.